src/library.js

```diff
@@ -453,7 +453,8 @@ LibraryManager.library = {
   clock__sig: 'i',
   clock: function() {
     if (_clock.start === undefined) _clock.start = Date.now();
-    return ((Date.now() - _clock.start) * ({{{ cDefine('CLOCKS_PER_SEC') }}} / 1000))|0;
+    let t = ((Date.now() - _clock.start) * ({{{ cDefine('CLOCKS_PER_SEC') }}} / 1000))|0;
```

Comment:
Sadly we can't use `let` yet in generated code.  Hopefully soon: #11984 .   In the mean time this has to be `var` I think.